docs/app-dev-context/spawning-activities.md

```diff
@@ -1,5 +1,5 @@
 Calls to spawn [Activity Executions](/concepts/what-is-an-activity-execution) are written within a [Workflow Definition](/concepts/what-is-a-workflow-definition).
-The call to spawn an Activity Execution generates the [ScheduleActivityTask](/references/commands/#scheduleactivitytask) Command.
+The call to spawn an Activity Execution generates the [ScheduleActivityTask](/references/commands/#scheduleactivitytask) command.
```

Comment:
However, [Command](https://docs.temporal.io/concepts/what-is-a-command/) is a Temporal term. (A Termporal?)